ERCS/erc-7590.md

```diff
@@ -0,0 +1,178 @@
+---
+eip: 7590
+title: ERC-20 Holder Extension for NFTs
+description: Extension to allow NFTs to receive and transfer ERC-20 tokens.
+author: Steven Pineda (@steven2308), Jan Turk (@ThunderDeliverer)
+discussions-to: https://ethereum-magicians.org/t/token-holder-extension-for-nfts/16260
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-01-05
+requires: 20, 165, 721
+---
+
+## Abstract
+
+This proposal suggests an extension to [ERC-721](./eip-721.md) to enable easy exchange of [ERC-20](./eip-20.md) tokens. By enhancing [ERC-721](./eip-721.md), it allows unique tokens to manage and trade [ERC-20](./eip-20.md) fungible tokens bundled in a single NFT.
```

Comment:
Your abstract should include enough information to give the reader a high-level technical overview of _how_ your proposal works.